app/controllers/users_controller.rb

```diff
@@ -55,4 +56,27 @@ def set_following_relationship
     end
   end
 
+  def charge_user_with_stripe
+    
+    # Set your secret key: remember to change this to your live secret key in production
+    # See your keys here https://dashboard.stripe.com/account/apikeys
+    Stripe.api_key = ENV['STRIPE_SECRET_KEY']
+
+    # Get the credit card details submitted by the form
+    token = params[:stripeToken]
+
+    # Create the charge on Stripe's servers - this will charge the user's card
+    begin
+      charge = Stripe::Charge.create(
+        :amount => 3000, # amount in cents, again
+        :currency => "usd",
+        :source => token,
+        :description => 'Signs up for #{@user.full_name}'
```

Comment:
This is not going to work like you hope. You need to use double quotes for interpolation to work.